### PR TITLE
[Compat]: Deeper and Darker Compat

### PR DESCRIPTION
**Pull request by JorisDeBeer**
:)

### CODE REVIEW

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]
Looks good! ~Echo planks needs to be changed to just echo for the recipe system to not throw a fit~.

We need to figure out an alternative name for the echo wood.

**Review by SigmundGranaas — APPROVED** [review]
Works well as long as #952 gets merged. Good job!

**Comment by SigmundGranaas** [comment]
Wait, we should add deeper_and_darker to the list of compatible mods in the curseforge and modrinth configuration: https://github.com/SigmundGranaas/forgero/blob/f8a5b5bd13d0feb036a1b1ef7988722946bc893b/fabric/build.gradle#L112-L122

**Comment by JorisDeBeer** [comment]
Could you invite me to the modrinth page too? 

Then I can add the new compats to the mod description page too

**Comment by SigmundGranaas** [comment]
You have been invited

**Comment by JorisDeBeer** [comment]
bro gave me zero permission to do anything lol 

Cant edit the description

**Comment by SigmundGranaas** [comment]
Redact this inaccurate statement

**Comment by JorisDeBeer** [comment]
I AM SORRY SIR

File: fabric/build.gradle [hunk 1]
```diff
@@ -119,6 +119,7 @@ curseforge {
 				optionalDependency "betternether"
 				optionalDependency "mystical-crops"
 				optionalDependency "mythicmetals"
+				optionalDependency "deeperdarker"
```
Comment (SigmundGranaas) [inline fabric/build.gradle:122]: Add this to the modrinth list as well.

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]
Small merge fix needs to be applied with the blaze rod fix, also the mod needs to be added to modrinth suggestions as well. Other than that, looks good to go!

**Review by SigmundGranaas — APPROVED** [review]
Looks good!

**Pull request merged**